styles/icomoon/demo-files/demo.js

```diff
@@ -0,0 +1,30 @@
+if (!('boxShadow' in document.body.style)) {
```

Comment:
are we using this?

Reply:
no, we are not using this it was removed